{"name": "test_invalid_login[edge]", "status": "passed", "description": "Verify invalid login shows error message.", "attachments": [{"name": "log", "source": "b5db0416-c0ba-4c8b-8de0-9955b3041fe0-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "setup", "value": "'edge'"}], "start": 1761139802651, "stop": 1761139806859, "uuid": "78b0e71e-356f-40aa-ba1f-f61574976ac5", "historyId": "2308ebe558d9e2a5c279a2694121ab80", "testCaseId": "896da2e0f49458712d4a62ebb5ef08cf", "fullName": "tests.test_keyword.TestKeywords#test_invalid_login", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_keyword"}, {"name": "subSuite", "value": "TestKeywords"}, {"name": "host", "value": "ASCENDI-FJC3FA0"}, {"name": "thread", "value": "6440-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_keyword"}], "titlePath": ["tests", "test_keyword.py", "TestKeywords"]}